services/portal/README.md

```diff
@@ -11,6 +11,7 @@ Rubin Science Platform portal aspect
 | affinity | object | `{}` | Affinity rules for the Portal pod |
 | config.cleanupInterval | string | `"36h"` | How long results should be retained before being deleted |
 | config.debug | string | `"FALSE"` | Set to `TRUE` to enable service debugging |
+| config.hipsUrl | string | `/api/hips/images/color_gri` in the local Science Platform | URL for default HiPS service |
```

Comment:
Has it been possible to test that `/api/hips/images/color_gri` actually works now - i.e., that the auth token is being passed along correctly?